apps/explorer/src/components/displaybox/DisplayBox.module.css

```diff
@@ -8,7 +8,7 @@ img.smallimage {
 
 div.modalcontainer,
 div.detailsbg {
-    @apply fixed top-0 left-0 z-20 h-full w-full;
+    @apply fixed left-0 top-0 z-20 h-full w-full;
```

Comment:
did you run a formatter for all of these CSS changes, or what's the context here? just curious

Reply:
Yeah updated prettier so these are just prettier changes